external/model-preparation-algorithm/configs/instance-segmentation/resnet50_maskrcnn/template_experimental.yaml

```diff
@@ -0,0 +1,47 @@
+# Description.
+model_template_id: ClassIncremental_Instance_Segmentation_MaskRCNN_ResNet50
+name: MaskRCNN-ResNet50-ClsIncr
+task_type: INSTANCE_SEGMENTATION
+task_family: VISION
+instantiation: "CLASS"
+summary: Class-Incremental Instance Segmentation for MaskRCNN-ResNet50
+application: ~
+
+# Algo backend.
+framework: MPA v1.1.0
+
+# Task implementations.
+entrypoints:
+  base: mpa_tasks.apis.detection.DetectionTrainTask
+  openvino: detection_tasks.apis.detection.OpenVINODetectionTask
+
+# Capabilities.
+capabilities: []
+
+# Hyperparameters.
+hyper_parameters:
+  base_path: ../configuration.yaml
+  parameter_overrides:
+    learning_parameters:
+      batch_size:
+        default_value: 8
```

Comment:
effinet-b0 template has 4 batch size as default while resnet50 has 8 bs. I think resnet50 is more heavier than effinet-b0, so more bs than effnet is not reasonable to me. Could you explain about the gpu memory usage?

Reply:
This should be modified through experiments, and it does not seem to be within the scope of this code review.

Reply:
I'm going to modify this through several settings with lr.

Reply:
First of all, the batch size has been modified to 4 to be aligned with the OTE model, and the specific LR will be adjusted during SC integration.